Podcast/EndpointRequest.swift

```diff
@@ -101,7 +101,7 @@ class EndpointRequest: Operation {
         return baseURLString + path
     }
     
-    func parameters() -> [String:Any] {
+    func parameters() -> [String:Any]? {
```

Comment:
What happens is you call parameters() in the request which ends up being [:] if there are none. However alamofire is expecting nil if there are node